tests/posix/pthread_pressure/Kconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023, Meta
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config TEST_NUM_CPUS
+	int "Number of CPUs to use in parallel"
+	range 1 MP_NUM_CPUS
+	default MP_NUM_CPUS
+	help
+	  The number of parallel threads to run during the test. The test
+	  thread itself yields so that all cores have some probability of
+	  causing racey behaviour.
+
+config TEST_DURATION_S
+	int "Number of seconds to run the test"
+	range 1 21600
+	default 1
```

Comment:
I'd bamp the default as we want to have this 'torture' test to be executed at least for some time. Something like 60 seconds looks more suitable here.

Reply:
60s would even be quite a lot for CI runs.

Any thoughts on that @stephanosio

Reply:
60s is fine. We have plenty of tests that take just as long or longer.